Formula/mediaconch.rb

```diff
@@ -18,6 +18,7 @@ class Mediaconch < Formula
   depends_on "sqlite"
   # fails to build against Leopard's older libcurl
   depends_on "curl" if MacOS.version < :snow_leopard
+  depends_on "libxslt" unless OS.mac?
```

Comment:
We don't take Linux-related changes to formulae here. Should be removed & forwarded onto Linuxbrew.